_pages/ato/walkthrough.md

```diff
@@ -45,21 +45,27 @@ The controls that are _not_ inherited from an underlying system must be listed i
 
 ### Step 4 — Assess the controls
 
-In other words, "verify that your system is secure". To do so, run the [security scans](../../security/scanning/).
+In other words, "verify that your system is secure". The first step in doing so is to run the [security scans](../../security/scanning/). This is a preliminary assessment, final assessment will be done in collaboration with the GSA Office of the Chief Security Officer (OCISO). 
 
-### Step 5 — Authorize the system
+### Step 5 - Complete documentation package
 
-The full list of data and functions in and of the system (in government parlance "mission based information types" and "delivery mechanisms") must be itemized in structured data. While the data types are obviously arbitrary and custom to each system we produce, the government has a formalized data set of mission functions that should be mapped to the system via NIST 800-60. For a Rails app, for example, this can simply be a link to the `db/schema.rb` file on GitHub.
+Fill out the [System Security Plan (SSP)](../ssp/) and link from your project's [checklist](../checklist/) issue. We hope to auto-create the SSP from the Compliance Masonry YAML file at some point, but for now you should copy and paste.
 
-Fill out the [System Security Plan (SSP)](../ssp/) and link from your project's [checklist](../checklist/) issue. The designated AO will conduct a quality assurance check on your materials.
+Fill out the Rules of Engagement and Project Plan documents, and link from your project's [checklist](../checklist/) issue. Work with your Infrastructure Lead to get the most up to date templates for these documents.
```

Comment:
Since the SSP and RoE are items in the checklist, it's redundant to describe them here - I say we just take these out of this section.